test/sampleData/arduino/arduino.ino

```diff
@@ -0,0 +1,2 @@
+void setup() {}
```

Comment:
Does this sketch need to include the peripheral driver files for them to be built?

Reply:
It seems to work without including them - the compiler is reading the files anyway (I verified that with the missing semicolon from issue #200)